docs/_docs/02_features/hooks.md

```diff
@@ -91,7 +91,18 @@ config {
 
 #### Configuration
 
-The `execute` parameter only accepts `tflint`, it will ignore any other parameter. Any desired extra configuration should be added in the `.tflint.hcl` file. It will work with a `.tflint.hcl` file in the current folder or any parent folder.
+By default, is executed internal `tflint` which will ignore any other parameter. Any desired extra configuration should be added in the `.tflint.hcl` file. It will work with a `.tflint.hcl` file in the current folder or any parent folder.
```

Comment:
```suggestion
By default, the internal `tflint` is executed which will ignore any other parameter. Any desired extra configuration should be added in the `.tflint.hcl` file. It will work with a `.tflint.hcl` file in the current folder or any parent folder.
```